vendor/engines/c2po/app/services/credit_card_account_builder.rb

```diff
@@ -0,0 +1,42 @@
+# Contains overrides for building a `CreditCardAccount` from params.
+# Dynamically called via the `AccountBuilder.for()` factory.
+class CreditCardAccountBuilder < AccountBuilder
+
+  protected
+
+  # Override strong_params for `build` account.
+  def account_params_for_build
```

Comment:
Is `for_build` always a superset of `for_update`? Would it be easier to maintain if we did `for_build` as `for_update + [....]`?